### DISCUSSION TITLE
Edit description title shouldn't say `Comment`

### DISCUSSION THREAD

**Discussion by jsoref** [community/community#44839] [Mobile] [answered]
GitHub Mobile beta

Open an issue

Select the edit item next to the description.

![Screenshot_20230119-183358.png](https://user-images.githubusercontent.com/2119212/213585646-1359f8e8-3d67-46ba-9934-cbfa65bfc9b0.png)

### Expected results

`Description`

**Comment by stevepopovich** [comment, marked as answer]
Hi @jsoref I passed this along to the team for consideration. Thanks!
